modules/shibboleth/templates/shibboleth2.xml.erb

```diff
@@ -65,6 +65,14 @@
                     <md:GivenName>Unity SSO Support</md:GivenName>
                     <md:EmailAddress>unity-sso@researchresearch.com</md:EmailAddress>
                 </md:ContactPerson>
+
+                <mdui:UIInfo>
+                    <mdui:DisplayName xml:lang="en">*Unity</mdui:DisplayName>
+                    <mdui:Description xml:lang="en">*Unity is the new way to work with anyone in universities and research. It’s a simple place to form groups, share files and work on them together.</mdui:Descriptione>
```

Comment:
But the opening tag is english (or french). One cannot simply be _half_ italian!